frontend/packages/ceph-storage-plugin/src/components/bucket-class/wizard-pages/namespace-store-pages/multi-namespace-store.tsx

```diff
@@ -23,22 +23,27 @@ export const MultiNamespaceStorePage: React.FC<MultiNamespaceStoreProps> = React
     const openModal = () => NamespaceStoreModal({ namespace });
     const [selectedCount, setSelectedCount] = React.useState(state.readNamespaceStore.length);
     const [enabledItems, setEnabledItems] = React.useState([]);
+    const [selectedItem, setSelectedItem] = React.useState(
+      getName(state.writeNamespaceStore[0]) ? getName(state.writeNamespaceStore[0]) : '',
+    );
 
     React.useEffect(() => {
       const readItems = state.readNamespaceStore.map(getName);
       setEnabledItems(readItems);
-    }, [state.readNamespaceStore]);
+    }, [state.readNamespaceStore, state.writeNamespaceStore, selectedItem]);
```

Comment:
```suggestion
    }, [state.readNamespaceStore]);
```
Why do we need to watch on writeNamespaceStore and selectedItem?
If that changes readNamespaceStore somehow then wouldn't readNamespaceStore just get retriggered?

Reply:
true. forgot to remove it , thanks